### ISSUE TITLE
Release package in melpa stable

### ISSUE THREAD

**Issue by favadi** [ericdanan/counsel-projectile#21]
Please tag a release so the package will be available in melpa stable: https://stable.melpa.org/.

**Comment by ericdanan** [comment]
Good idea.

I just updated the required packages versions. To be sure I will wait a few days and if no issue appears I will bump the counsel-projectile version to 0.2 and tag a release.

**Comment by favadi** [comment]
@ericdanan do you think it is ready to tag a release?

**Comment by ericdanan** [comment]
There are a couple of other issues and PR pending but they do not really block a stable release IMO. However there was a recent change in the way projectile switches project and there is a pending issue in the projectile repo that is perhaps related to this. So I'd rather take some time to see what happens with this issue and test whether counsel-projectile is affected.

**Comment by ericdanan** [comment]
Hi, sorry to be so long with this, but the projectile change is indeed causing issues (see #32 ) which should be fixed before tagging a stable release.

**Comment by thblt** [comment]
Do not take this badly, but *please*, *please* do not tag an initial release if you don't intend to create tags at *every single* bugfix and notable improvement.  When stable Emacs was 24.x, a lot of people did create release tags for their packages so that people who wished to use only Stable could use them.  In a lot of cases, there were virtually no releases beyond 0.1 or 1.0, stable versions were lagging way behind the continuous-integration Melpa versions, weird bugs and "please tag a new release" issues started proliferating.  With Emacs 25.x, it has become possible to assign priorities to archive sources, so package.el will download packages from Melpa Stable if they're available there, and let their Emacs fallback to Melpa if they aren't.  This is a better solution than being stuck with an obsolete 0.1. 

This is not an argument against tagged releases in general, but the consequences deserve to be considered.

**Comment by ericdanan** [comment]
Sorry for not replying earlier!

I'm willing to tag a release and try to stick to doing it consistently. By now I've adressed the projectile issues that affected project switching, and I think the current code is stable enough (perhaps except for new commits I made today, but I could always tag before these if necessary). But, it depends on ivy changes that are posterior to the last tagged release (0.9.1), so I guess I have to wait for the next ivy release.

**Comment by mdorman** [comment]
There's been a new release of `ivy`, maybe it's time?

**Comment by ericdanan** [comment]
Yes it is. As you perhaps saw, following the release of ivy 0.10 I pushed some changes to counsel projectile to try making it more user friendly and bumped the version to 0.2. I am just waiting a bit for users to test in case I had introduced new issues (such as #65 that was found today). If there is no blocking issue by the end of next week I will tag a release so melpa stable can pick it up. Thanks for your patience.
